sound/soc/sof/intel/intel_ext_manifest.h

```diff
@@ -0,0 +1,34 @@
+/* SPDX-License-Identifier: ((GPL-2.0 WITH Linux-syscall-note) OR BSD-3-Clause) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2020 Intel Corporation. All rights reserved.
+ */
+
+/*
+ * Intel extended manifest is a extra place to store intel specific
```

Comment:
s/intel/Intel/

Reply:
why intel twice in the header name? Its already in the intel folder no? so ext_manifest.h is enough?

Reply:
I tried to avoid same file name, ext_manifest.h  is in include/sound/sof/. not a big deal. Should I rename it to ext_mainfest.h